libraries/AP_RCProtocol/AP_RCProtocol_CRSF.h

```diff
@@ -43,7 +43,11 @@ class AP_RCProtocol_CRSF : public AP_RCProtocol_Backend {
     bool change_baud_rate(uint32_t baudrate);
     // bootstrap baudrate
     uint32_t get_bootstrap_baud_rate() const {
+#if AP_RC_CHANNEL_ENABLED
```

Comment:
The define makes no sense to me here. Maybe the name is wrong.

Reply:
Well, the library is `RC_Channel` - as in `libraries/RC_Channel`

The class name is `RC_Channel`.

We should prefix everything with `AP`.  Thus `AP_RC_CHANNEL_ENABLED`.